pkg/server/middleware/handler_utils.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2021 Storj Labs, Inc.
```

Comment:
nit: copyright should be 2022.

Maybe we can also call these files `requestid`? `handler_utils` is probably a bit too generic.